indico/modules/rb/controllers/backend/admin.py

```diff
@@ -503,3 +504,45 @@ def _process_POST(self, args):
         db.session.flush()
         RHRoomsPermissions._jsonify_user_permissions.clear_cached(session.user)
         return jsonify(id=room.id)
+
+
+_base_schema = {
```

Comment:
what about using actual `mm.Schema` subclasses here and then proper inheritance below instead of `dict(.., **..)`?

Reply:
Is it really worth it to create a class here? :P